frontend/components/language/Selector.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+type GenericLang = 'en' | 'de' | 'fr' | 'es'
```

Comment:
I'd put this type, and maybe the whole `options` list below somewhere in `@/lib`, I can see this being a thing that gets copied around if not put somewhere meaningful and easily importable.

Reply:
Good idea, done, in a way that is reusable for the next PR :)